ext/authoring/services/src/main/java/org/exoplatform/services/wcm/extensions/scheduler/impl/ChangeStateCronJobImpl.java

```diff
@@ -164,43 +166,49 @@ private long changeStateForNodes(SessionProvider sessionProvider, String propert
     
     for (NodeIterator iter = queryResult.getNodes(); iter.hasNext();) {
       Node node_ = iter.nextNode();
-      
       String path = node_.getPath();
-      if (!path.startsWith("/jcr:system")) {
-        if (NORMAL_NODE == nodeType) {
-          Date nodeDate = node_.getProperty(property).getDate().getTime();
-          if (LOG.isInfoEnabled()) LOG.info("'" + toState + "' " + node_.getPath() + " (" + property + "="
-              + format.format(nodeDate) + ")");
-  
-          if (PublicationDefaultStates.UNPUBLISHED.equals(toState)) {
-            if (node_.hasProperty(AuthoringPublicationConstant.LIVE_REVISION_PROP)) {
-              String liveRevisionProperty = node_.getProperty(AuthoringPublicationConstant.LIVE_REVISION_PROP)
-              .getString();
-              if (!"".equals(liveRevisionProperty)) {
-                Node liveRevision = session.getNodeByUUID(liveRevisionProperty);
-                if (liveRevision != null) {
-                  context_.put(AuthoringPublicationConstant.CURRENT_REVISION_NAME,
-                      liveRevision.getName());
+      try {
+        if (!path.startsWith("/jcr:system") && !trashService.isInTrash(node_)) {
+          if (NORMAL_NODE == nodeType) {
+            Date nodeDate = node_.getProperty(property).getDate().getTime();
+            if (LOG.isInfoEnabled()) LOG.info("'" + toState + "' " + node_.getPath() + " (" + property + "="
+                    + format.format(nodeDate) + ")");
+
+            if (PublicationDefaultStates.UNPUBLISHED.equals(toState)) {
+              if (node_.hasProperty(AuthoringPublicationConstant.LIVE_REVISION_PROP)) {
+                String liveRevisionProperty = node_.getProperty(AuthoringPublicationConstant.LIVE_REVISION_PROP)
+                        .getString();
+                if (!"".equals(liveRevisionProperty)) {
+                  Node liveRevision = session.getNodeByUUID(liveRevisionProperty);
+                  if (liveRevision != null) {
+                    context_.put(AuthoringPublicationConstant.CURRENT_REVISION_NAME,
+                            liveRevision.getName());
+                  }
                 }
               }
+
             }
-  
+            publicationPlugin.changeState(node_, toState, context_);
+            ret++;
+          } else {
+            if (LOG.isInfoEnabled()) LOG.info("'" + toState + "' " + node_.getPath());

```

Comment:
```suggestion
            LOG.info("'{}' {}", toState, node_.getPath());
```
Is it possible to use this format instead ?